OWNERS

```diff
@@ -0,0 +1,8 @@
+## ADMINS: People who control settings for the repo.
+adrianchiris
+dougbtv
+
+## Maintainers: People who can merge code in this repo.
+zeeke
+SchSeba
+Eoghan1232
```

Comment:
I'm really looking for feedback on who should take these roles. 

The current proposal is based on some simple ideas:
- removing roles the NPWG doesn't define in the governance doc
- remove folks who haven't contributed as a maintainer or admin for more than two years
- restrict admins to folks who are also NPWG org owners